check_warnings.sh

```diff
@@ -6,16 +6,12 @@
 # Desc: Check for warnings during build process of repo $CI_SOURCE_PATH
 
 packages_with_warnings() {
-   for pkg in $(catkin_topological_order $CI_SOURCE_PATH --only-names 2> /dev/null) ; do
+   for pkg in $(colcon info | grep 'name: ' | sed -e "s/.*name: //g" 2> /dev/null) ; do
       # Warnings manifest themselves log files in catkin tools' logs folder
-      files=$(find $ROS_WS/logs/$pkg -name "*build.cmake.000.log.stderr" -o -name "*build.make.00[01].log.stderr" 2> /dev/null)
-      # Extract types of failures from file names
-      issues=""
-      issues="${issues}$(echo $files | sed -ne 's:.*/build\.cmake\.000.*:cmake :p')"
-      issues="${issues}$(echo $files | sed -ne 's:.*/build\.make\.000.*:build :p')"
-      issues="${issues}$(echo $files | sed -ne 's:.*/build\.make\.001.*:test-build :p')"
+      log_file=$(find $ROS_WS/log/latest_build/$pkg -name "stderr.log" 2> /dev/null)
+      # Extract the stderr.log file
```

Comment:
Where is `stderr.log` being extracted? This code only prints the log file name instead of the warnings, right?
Also shouldn't the log and errors be the same so we can port this code directly?

Reply:
> Where is stderr.log being extracted? This code only prints the log file name instead of the warnings, right?

Comment has been updated.

> Also shouldn't the log and errors be the same so we can port this code directly?

I'm not sure I understand this

Reply:
Nevermind. I didn't realize that the log files are indeed different. I thought we could keep the issue extraction part.